tiled/server/schemas.py

```diff
@@ -396,6 +397,7 @@ class Principal(pydantic.BaseModel):
     api_keys: List[APIKey] = []
     sessions: List[Session] = []
     latest_activity: Optional[datetime] = None
+    access_token: Optional[str] = None
```

Comment:
Shouldn't the principal be identifiable from the access token, rather than holding the latest(?) access token?